pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [build-system]
-requires = ["wheel"]
-build-backend = "setuptools.build_meta"
+requires=["wheel"]
```

Comment:
Nitpick: why the removal of spaces around `=`? (which differs from the rest of the file).

Reply:
Oops, this is an accident, let me fix it.

Reply:
Fixed.